.github/workflows/blui-ci.yml

```diff
@@ -0,0 +1,287 @@
+    branches:
+      - '*/*'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  prettier_lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+    - run: yarn install --frozen-lockfile
+      working-directory: components
+    - run: yarn prettier
+      working-directory: components
+    - run: yarn lint
+      working-directory: components
+    - run: yarn install --frozen-lockfile
+      working-directory: demos/api
+    - run: yarn prettier
+      working-directory: demos/api
+    - run: yarn lint
+      working-directory: demos/api
+
+  unit_test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+    - run: yarn install --frozen-lockfile
+      working-directory: components
+    - run: yarn coverage
+      working-directory: components
+    - name: Upload to Codecov
+      uses: codecov/codecov-action@v3
+      with:
+        directory: ./components/coverage/
+        files: cobertura-coverage.xml
+        flags: unittests
+        name: codecov-report
+        verbose: true
+
+  build_library:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+    - run: yarn --frozen-lockfile
+      working-directory: components
+    - run: yarn build
+    - name: Save build
+      uses: actions/upload-artifact@v3
+      with:
+        name: dist
+        if-no-files-found: error
+        path: dist
+
+  test_artifacts:
+    runs-on: ubuntu-latest
+    needs: build_library
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Download the dist folder
+      uses: actions/download-artifact@v3
+      with:
+        name: dist
+        path: dist
+    - run: yarn test:artifacts
+
+  npm_pack:
+    runs-on: ubuntu-latest
+    needs: build_library
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Download the dist folder
+      uses: actions/download-artifact@v3
+      with:
+        name: dist
+        path: dist
+    - run: npm pack
+      working-directory: dist
+    - name: Save npm pack
+      uses: actions/upload-artifact@v3
+      with:
+        name: tarball
+        if-no-files-found: error
+        path: dist
+
+  build_showcase:
```

Comment:
Do we need `build_showcase` in this or that should be in the showcase repo?

Reply:
It can be either but this one is using the RN component library package from tarball to build showcase....